### PR TITLE
Add more example code and an ephemeral user tutorial

### PR DESCRIPTION
**Pull request by rosew**

### CODE REVIEW

File: docs/tutorials/02_ephemeral_user.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
+Please see the Hello World Tutorial for initial setup of the Forsta Messenger Client. Complete code for this tutorial is available at:
```
Comment (mayfield) [inline docs/tutorials/02_ephemeral_user.md:1]: Link to Hello World tutorial would be nice here.

File: docs/tutorials/02_ephemeral_user.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    const myClient = new forsta.messenger.Client(document.getElementById('my-messenger'),
+      { orgEphemeralToken: 'TESTING' },
+      { });
+  </script>
+</html>
+
+```
+
+The Forsta Messenger Client includes several callbacks that can be configured when the
+client is initialized. In this case we will set the onLoaded callback to do the following:
```
Comment (mayfield) [inline docs/tutorials/02_ephemeral_user.md:35]: backticks on the "onLoaded" a la., `onLoaded` will render that code ref as a sample block as is common when making a coding ref.

File: docs/tutorials/02_ephemeral_user.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+- Create a new conversation thread with a tagged users
+- Open the thread in the client
+- Display the client once the thread is available
+
+First, create the onload callback. It uses the threadMake command to generate a new thread
+between the ephemeral user and one or more users defined by their tags. For more information
+on using tags see {@link https://docs.forsta.io/docs/tag-expressions}. Once a thread has been
```
Comment (mayfield) [inline docs/tutorials/02_ephemeral_user.md:43]: Bonus points if that link could be made an `External` ref like the `ExchangePayload` and then we could convert any refs to `TagExpression` (and this one) to point to the external ref which would go to that link.  This way all internal refs to tag expression could be funneled through the same flow of "Common name" -> "Locally defined description of tag expression with link to detailed docs" -> "click away to external docs on full tag expression info".

File: docs/tutorials/02_ephemeral_user.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+created, it opens the thread in the client window and changes the visibility on the div to be visible.
+
+```html
+  async function onLoaded(client) {
+    thread_id = await client.threadMake("@support");
```
Comment (mayfield) [inline docs/tutorials/02_ephemeral_user.md:48]: nit: snakecase isn't considered javascript-esc.  You find internal refs using snake case, and my examples will tend to be JS idiomatic, so might do well to use the same style.

File: docs/tutorials/02_ephemeral_user.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    await client.threadOpen(thread_id);
+      
+    var messenger = document.getElementById("my-messenger");
```
Comment (mayfield) [inline docs/tutorials/02_ephemeral_user.md:51]: nit: `const` and `let` replaced `var` for modern JS variable declaration..  I can go into detail why if you'd like, but it's a good habit to switch to `const` for everything and `let` only when you really know what you're doing.

File: docs/tutorials/index.json [hunk 6]
```diff
@@ -0,0 +1,8 @@
+{
+  "01_hello_world": {
+    "title": "Hello World"
+  },
+  "02_ephemeral_user": {
+    "title": "Ephemeral User"
+  }
+}
```
Comment (mayfield) [inline docs/tutorials/index.json:8]: Nice,  good find.

**Review by mayfield — APPROVED** [review]
I left several comments, but they're all low prio.  Feel free to make any of those changes in subsequent phases and merge this one whenever you like.

Thanks @rosew !

File: docs/tutorials/02_ephemeral_user.md (hunk 4 above)
Comment (rosew) [reply docs/tutorials/02_ephemeral_user.md:48]: Thank you! Nits are good! My Javascript is all self taught so I appreciate the lessons to clean up my style. My rails style is bleeding through where it shouldn't.

File: docs/tutorials/02_ephemeral_user.md (hunk 3 above)
Comment (rosew) [reply docs/tutorials/02_ephemeral_user.md:43]: Done but will be in the next round of docs.

**Comment by rosew** [comment]
Thank you, all comments done but will go out with tutorial 3 for regular users.

**Pull request merged**